docs/schema.md

```diff
@@ -382,6 +382,8 @@ The defined ecosystems are:
 | `Debian` | The Debian package ecosystem; the `name` is the name of the source package. The ecosystem string might optionally have a `:<RELEASE>` suffix to scope the package to a particular Debian release. `<RELEASE>` is a numeric version specified in the [Debian distro-info-data](https://debian.pages.debian.net/distro-info-data/debian.csv). For example, the ecosystem string "Debian:7" refers to the Debian 7 (wheezy) release.  |
 | `Hex` | The package manager for the Erlang ecosystem; the `name` is a Hex package name.  |
 | `Android`  | The Android ecosystem; the `name` field is the Android component name that the patch applies to, as shown in the [Android Security Bulletins](https://source.android.com/security/bulletin) such as `Framework`, `Media Framework` and `Kernel Component`. The exhaustive list of components can be found at the [Appendix](#android-ecosystem-components). |
+| `GitHub Actions` | The GitHub Actions ecosystem; the `name` field is an action name. |
+| `Pub` | The package manager for the Dart ecosystem; the `name` field is a Dart package name. |
```

Comment:
@jonasfj Does this seem good to you? Are there any normalization rules etc for Pub packages?

Reply:
This looks solid to me.

There is some corner case around custom pub repositories. But that's probably solved with a `purl` that has a `?repository_url=...` qualifier.